### ISSUE TITLE
Add vc plugin matrix operations

### ISSUE THREAD

**Issue by niermann999** [acts-project/algebra-plugins#53]
Currently, there are no useful Vc native descriptions of matrices or matrix operations. Some care is going to be needed to treat matrix dimensions and e.g. floating point precision, underlying vector width etc.

**Comment by beomki-yeo** [comment]
Do we want to keep this open?

**Comment by niermann999** [comment]
Yes, please. I don't think we should remove the vc plugin just yet. I will try to make it work as soon as I have the time...

**Comment by beomki-yeo** [comment]
ok

**Comment by niermann999** [comment]
Matrix operations are available now
